### DISCUSSION TITLE
NoScribeEditor - Unable to use Qtawesome

### DISCUSSION THREAD

**Discussion by Datex-exo** [kaixxx/noScribe#81] [Q&A] [answered]
Hello, the NoScribe software is very impressive. However, I am having trouble with NoScribe Editor on my company workstations due to font restrictions that cannot be removed. (Windows/fonts only and therefore unable to use Qtawesome) Is it possible to have the NoScribe software alone, that is without the Editor application so we can install it on our Windows workstations without errors ?

**Comment by kaixxx** [comment]
The installer of noScribe is very simple: It copies all the files into the installation directory (usually C:\Program Files (x86)\noScribe) and creates shortcuts in the windows start menu to noScribe.exe and noScribeEdit.exe. 
You can replicate this with your own script: Copy all the files from a working installation but leave noScribeEditor out (the shortcut and the sudirectory "noScribeEdit"). 
Your script should also create the file "config.yml" in "C:\Users\<USERNAME>\AppData\Local\noScribe\noScribe\" with the following line:
`auto_edit_transcript: 'False'`
This prevents the editor from starting automatically after the transcript is finished. 

Now, the only problem left is that users will get an error message if they click on the editor-button in the lower right. I think this is a minor issue.

**Reply by Datex-exo** [reply]
Thank you very much for the quick response. I’ll proceed like that. If however there is a way to remove the icon in the bottom right I am interested to know the manipulation to do. Thanks again for this application !

**Reply by kaixxx** [reply, marked as answer]
You can remove the button in the source code. But then you need to recompile the app with pyinstaller. Everything you need is in this repo. But you need a little bit of experience with python and pyinstaller to get it working. 
If you start digging into the source code, I would rather try to get qtawesome working with fonts that are installed in windows/fonts. I had a quick try: It is not as easy as just moving the fonts to windows/fonts, but there might be other options. 
Anyhow, it all depends on how much time and effort you want to put into this.

**Reply by Datex-exo** [reply]
All right, I’ll work on it. Thank you very much
